bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncTable.java

```diff
@@ -137,11 +141,17 @@ public CheckAndMutateBuilder checkAndMutate(byte[] row, byte[] family) {
 
     private final CheckAndMutateUtil.RequestBuilder builder;
     private final BigtableDataClient client;
+    // Once the IBigtableDataClient interface is implemented this will be removed
+    protected final RequestContext requestContext;
 
     public CheckAndMutateBuilderImpl(BigtableDataClient client, HBaseRequestAdapter hbaseAdapter,
         byte[] row, byte[] family) {
       this.client = client;
       this.builder = new CheckAndMutateUtil.RequestBuilder(hbaseAdapter, row, family);
+      BigtableTableName bigtableTableName = hbaseAdapter.getBigtableTableName();
```

Comment:
as a next step, please add a new method in `BigtableTableName` to create a gcb `InstanceName`